CefSharp/Internals/JavascriptObjectRepository.cs

```diff
@@ -404,6 +411,8 @@ internal bool TrySetProperty(long objectId, string name, object value, out strin
         /// <param name="camelCaseJavascriptNames">camel case the javascript names of properties/methods</param>
         private void AnalyseObjectForBinding(JavascriptObject obj, bool analyseMethods, bool analyseProperties, bool readPropertyValue, bool camelCaseJavascriptNames)
         {
+            var isWindowPropertyFirstCharacterLowercase = StringCheck.IsFirstCharacterLowercase(WindowPropertyName);
+
```

Comment:
Please revert this set of changes. My previous comment was simply to reduce code duplication, I meant extract the core part of the function into say StringCheck(we'll need to me StringCheck to be more generic)

Reply:
Noted. reverted back